docs/en/pid-issuing.rst

```diff
@@ -0,0 +1,511 @@
+
+    client_id=$thumprint-of-the-jwk-in-the-cnf-wallet-attestation$
+    &grant_type=authorization_code
+    &code=SplxlOBeZQQYbYS6WxSbIA
+    &redirect_uri=eudiw://start.wallet.it
+    &code_verifier=dBjftJeZ4CVP-mB92K27uhbUJU1p1r_wW1gFWFOEjXk
+    &client_assertion_type=urn:ietf:params:oauth:client-assertion-type:jwt-bearer
+    &client_assertion=$WalletInstanceAttestation$
+
+
+**Step 15:** The PID Provider validates the request and if it is successful, it issues an *access token* (bound to the DPoP key) and a fresh *c_nonce*.  
+
+.. code-block::
+
+    HTTP/1.1 200 OK
+    Content-Type: application/json
+    Cache-Control: no-store
+    
+    {
+    "access_token": "Kz~8mXK1EalYznwH-LC-1fBAo.4Ljp~zsPE_NeO.gxU",
+    "token_type": "DPoP",
+    "expires_in": 2677,
+    "c_nonce": "tZign[...]snFbp",
+    "c_nonce_expires_in": 86400
+    }
+
+
+**Steps 16-18:** The Wallet Instance creates a fresh key pair the new credential shall be bound to.  Then, it creates a proof of possession with the new key and the *c_nonce* obtained in **Step 15** and it creates a DPoP proof for the request to the PID credential issuance endpoint.  
+
+**Step 19:** The Wallet Instance sends a PID issuance request to the PID Provider credential endpoint. It contains the *access token*, the *DPoP proof*, the *credential type*, the *proof* (proof of possession of the key) and the *format*.
+
+.. note::
+
+    **PID Credential Schema and Status registration:**  The PID Provider MUST register all the PID credential related information for the later status verification and revocation. 
+
+.. code-block::
+
+    POST /credential HTTP/1.1
+    Host: pid.it
+    Content-Type: application/x-www-form-urlencoded
+    Authorization: DPoP Kz~8mXK1EalYznwH-LC-1fBAo.4Ljp~zsPE_NeO.gxU
+    DPoP: eyJ0eXAiOiJkcG9wK2p3dCIsImFsZyI6IkVTMjU2IiwiandrIjp7Imt0eSI6Ik
+        VDIiwieCI6Imw4dEZyaHgtMzR0VjNoUklDUkRZOXpDa0RscEJoRjQyVVFVZldWQVdCR
+        nMiLCJ5IjoiOVZFNGpmX09rX282NHpiVFRsY3VOSmFqSG10NnY5VERWclUwQ2R2R
+        1JEQSIsImNydiI6IlAtMjU2In19.eyJqdGkiOiJlMWozVl9iS2ljOC1MQUVCIiwiaHRtIj
+        oiR0VUIiwiaHR1IjoiaHR0cHM6Ly9yZXNvdXJjZS5leGFtcGxlLm9yZy9wcm90ZWN0Z
+        WRyZXNvdXJjZSIsImlhdCI6MTU2MjI2MjYxOCwiYXRoIjoiZlVIeU8ycjJaM0RaNTNF
+        c05yV0JiMHhXWG9hTnk1OUlpS0NBcWtzbVFFbyJ9.2oW9RP35yRqzhrtNP86L-Ey71E
+        OptxRimPPToA1plemAgR6pxHF8y6-yqyVnmcw6Fy1dqd-jfxSYoMxhAJpLjA
+    
+    credential_definition=%7B%22type%22:%5B%22eu.eudiw.pid.it%22%5D%7D
+    &format=vc+sd-jwt
+    &proof=%7B%22proof_type%22:%22...-ace0-9c5210e16c32%22%7D
+
+
+
+A non-normative example of proof parameter is given below:
+
+.. code-block:: JSON
+
+    {
+    "proof_type": "jwt",
+    "jwt": "eyJraWQiOiJkaWQ6ZXhhbXBsZTplYm …"
+    }
+
+
+Where the JWT looks like this:
+
+.. code-block:: JSON
+
+    {
+    "alg": "ES256",
+    "typ": "openid4vci-proof+jwt",
+    "jwk": <…> 
+    }
+    .
+    {
+    "iss": "0b434530-e151-4c40-98b7-74c75a5ef760",
+    "aud": "https://pid.it",
```

Comment:
please, use example.org in non-normative examples

Reply:
done